tests/product/tests/test_rules.py

```diff
@@ -2,45 +2,136 @@
 Kubernetes CIS rules verification.
 This module verifies correctness of retrieved findings by manipulating audit and remediation actions
 """
+import datetime
 import time
 
 import pytest
 from commonlib.io_utils import get_logs_from_stream, get_k8s_yaml_objects
 from pathlib import Path
-
+from product.tests.tests.file_system.file_system_test_cases import *
```

Comment:
is there a better way to import it?

Reply:
for now, now, let's have a look at it again after  the changes in your PR